.vscode/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "0.1.0",
+    "command": "npm",
+    "isShellCommand": true,
+    "tasks": [
+        {
+            "suppressTaskName": true,
+            "taskName": "Start Node Foreman in Dev",
+            "args": [
+                "run",
+                "dev"
+            ],
+            "isWatching": true,
+            "isBuildCommand": true
+        }
+    ]
+}
```

Comment:
This _lets_ you run the project through VS Code if you choose - it is optional, just runs `npm run dev` and keeps results inline.